service-api/app/src/App/src/Service/Lpa/ValidateOlderLpaRequirements.php

```diff
@@ -23,49 +26,49 @@ class ValidateOlderLpaRequirements
 
     private DateTimeImmutable $earliestDate;
     private LoggerInterface $logger;
+    private FeatureEnabled $featureEnabled;
 
     /**
      * ValidateOlderLpaRequirements constructor.
      *
      * @param LoggerInterface $logger
      * @codeCoverageIgnore
      */
-    public function __construct(LoggerInterface $logger)
+    public function __construct(LoggerInterface $logger, FeatureEnabled $featureEnabled)
     {
         $this->logger = $logger;
         $this->earliestDate = new DateTimeImmutable(self::EARLIEST_REG_DATE);
+        $this->featureEnabled = $featureEnabled;
     }
 
     /**
      * @param array $lpa An LPA data structure
-     *
-     * @return bool The LPA has valid requirements
      * @throws Exception Thrown when unable to parse LPA registration date as a date
      */
-    public function __invoke(array $lpa): bool
+    public function __invoke(array $lpa): void
     {
-        if ($lpa['status'] !== self::NECESSARY_STATUS) {
+        if (!($lpa['status'] === self::NECESSARY_STATUS)) {
```

Comment:
```suggestion
        if ($lpa['status'] !== self::NECESSARY_STATUS) {
```
!(===) is a strange way of just writing !==

Reply:
:D yes, Thanks Gemma